src/router.ts

```diff
@@ -120,7 +124,7 @@ class Router {
             dipDeposit.strikeUsdcPerToken,
           )
         ] = dipDeposit;
-        return;
+        return routedSize;
```

Comment:
fine how you did it, but I would changed these to

return NO_ROUTED_SIZE;

and then the only that isnt would be return quantityTrade